src/jobs/quality_gate_job.yml

```diff
@@ -14,6 +14,9 @@ parameters:
   cache_path:
     type: string
     default: ""
+  working_directory:
+    type: string
+    default: ""
```

Comment:
Jobs already have working_directory parameter. See: https://circleci.com/docs/2.0/configuration-reference/#jobs
Rename it to void conflict.